node/silkworm/common/log.cpp

```diff
@@ -105,6 +126,7 @@ void BufferBase::flush() {
         }
         *file_ << line << std::endl;
     }
+    out_lck.unlock();
```

Comment:
Unnecessary (will be unlocked by the destructor)

Reply:
Yeah right ... nonetheless it helped my readability.
Like tons of `else`s we have here and there
Will remove

Reply:
Solved by 9e8dd04